### PR TITLE
Add non-SDK Python code example for Bulk Subscribe

### LINKED ISSUE #3321: [Bulk Subscribe] Need Python non-SDK code snippet
**What content needs to be created or modified?**
Python SDK example of bulk subscribe

**Where should the new material be placed?**
https://docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-bulk/#example-1

**The associated pull request from dapr/dapr, dapr/components-contrib, or other Dapr code repos**
dapr/python-sdk#464

### PR DESCRIPTION
**Pull request by hhunter-ms**
## Description

Add non-SDK python example for bulk subscribe

## Issue reference

PR will close: #3321

### CODE REVIEW

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md [hunk 1]
```diff
@@ -471,7 +470,59 @@ public class BulkMessageController : ControllerBase
...
+
+@app.route('/checkout', methods=['POST'])
+def checkout():
+    messages = request.json
+    for message in messages:
+        print(f"Received message: {message}")
+    return '', 200
+
+if __name__ == '__main__':
+    app.run(port=5000)
```
Comment (elena-kolevska) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md]:
```suggestion
import json
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/dapr/subscribe', methods=['GET'])
def subscribe():
    # Define the bulk subscribe configuration
    subscriptions = [{
        "pubsubname": "pubsub",
        "topic": "TOPIC_A",
        "route": "/checkout",
        "bulkSubscribe": {
            "enabled": True,
            "maxMessagesCount": 3,
            "maxAwaitDurationMs": 40
        }
    }]
    print('Dapr pub/sub is subscribed to: ' + json.dumps(subscriptions))
    return jsonify(subscriptions)


# Define the endpoint to handle incoming messages
@app.route('/checkout', methods=['POST'])
def checkout():
    messages = request.json
    print(messages)
    for message in messages:
        print(f"Received message: {message}")
    return json.dumps({'success': True}), 200, {'ContentType': 'application/json'}

if __name__ == '__main__':
    app.run(port=5000)

```

**Review by elena-kolevska — DISMISSED** [review]
The example didn't work for me when I ran it, @hhunter-ms were you able to get it to run? I modified it a bit and I'm able to see messages on the subscriber side now, but feel free to ignore if the original version works.

**Comment by hhunter-ms** [comment]
@elena-kolevska taking your example! Thank you!

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**